assets/js/state/sagas/index.js

```diff
@@ -119,7 +119,9 @@ function* initialDataFetch() {
   yield put(stopHostsLoading());
 
   yield put(startClustersLoading());
-  const { data: clusters } = yield call(get, '/clusters');
+  const { data: clusters } = yield call(get, '/clusters', {
```

Comment:
I guess for now this is enough. 
However at a certain point I'd find it of great help having something like `yield call(getV2, '/clusters');`
Feel free to keep it as you did for now.

Reply:
Fair enough. I wanted to keep the current changes minimalist.
This wouldn't require many changes, simply to add 4 new functions (get, post, del, put) with that naming.
Let's keep it on the radar